lib/gamera/modules/visitable.rb

```diff
@@ -8,7 +8,7 @@ module Visitable
     #
     # @param fail_on_redirect [Boolean] Whether to fail if the site redirects to a page that does not match the url_matcher regex
     # @raise [WrongPageVisited] if the site redirects to URL that doesn't match the url_matcher regex and fail_on_redirect is true
-    def visit(fail_on_redirect = true)
+    def visit(fail_on_redirect: true)
```

Comment:
Seems like this change should go in a `CHANGES.md` or at least a readme?  Then again, it's pre-beta software.